RunningOnDiRAC.md

```diff
@@ -0,0 +1,29 @@
+# Running On DiRAC
+
+Follow these instructinos to get an account (for the SCRC Project):
+
+    https://github.com/ScottishCovidResponse/SCRCIssueTracking/wiki/DiRAC-Access
+
+You should receive an email describing how to connect to one of the CSD3 login nodes,
+with something like (replacing your-user-name with your username):
+
+    ssh -l your-user-name login.hpc.cam.ac.uk
+
+Load Java:
+
+    module load jdk-8u141-b15-intel-17.0.4-3g6rhr2
```

Comment:
Might be worth noting somewhere that you need to do the PATH setting and module load each time you log in (but that you don't need to checkout the code each time)